src/node_types.cc

```diff
@@ -11,47 +13,60 @@ using v8::Value;
-  V(SharedArrayBuffer)                                                        \
-  V(Proxy)                                                                    \
-  V(ModuleNamespaceObject)                                                    \
-
-
-#define V(type) \
-  static void Is##type(const FunctionCallbackInfo<Value>& args) {             \
-    args.GetReturnValue().Set(args[0]->Is##type());                           \
-  }
+#define VALUE_METHOD_MAP(V)                                                    \
+  V(External)                                                                  \
+  V(Date)                                                                      \
+  V(ArgumentsObject)                                                           \
+  V(BigIntObject)                                                              \
+  V(BooleanObject)                                                             \
+  V(NumberObject)                                                              \
+  V(StringObject)                                                              \
+  V(SymbolObject)                                                              \
+  V(NativeError)                                                               \
+  V(RegExp)                                                                    \
+  V(AsyncFunction)                                                             \
+  V(GeneratorFunction)                                                         \
+  V(GeneratorObject)                                                           \
+  V(Promise)                                                                   \
+  V(Map)                                                                       \
+  V(Set)                                                                       \
+  V(MapIterator)                                                               \
+  V(SetIterator)                                                               \
+  V(WeakMap)                                                                   \
+  V(WeakSet)                                                                   \
+  V(ArrayBuffer)                                                               \
+  V(DataView)                                                                  \
+  V(SharedArrayBuffer)                                                         \
+  V(Proxy)                                                                     \
+  V(ModuleNamespaceObject)
 
-  VALUE_METHOD_MAP(V)
+#define V(type)                                                                \
+  static void Is##type(const FunctionCallbackInfo<Value>& args) {              \
+    args.GetReturnValue().Set(args[0]->Is##type());                            \
+  }                                                                            \
+  static bool Is##type##FastApi(Local<Value> unused, Local<Value> receiver) {  \
```

Comment:
Isn't `receiver` the name that we should give to the first parameter if we used it?